### DISCUSSION TITLE
Using SIMATIC AX with the Software Controller

### DISCUSSION THREAD

**Discussion by RauerJ** [simatic-ax/.discussions#3] [Q&A] [answered]
The Software Controller is an S7-1500 runtime.
1) Is it possible to download a program from SIMATIC AX to the Software Controller?
2) Can I use AX2TIA-Blocks in TIA Portal in a Software Controller?

**Comment by theBadT** [comment, marked as answer]
As for now, this is not possible, as we only support S7 1500 PLCs.

However, we've already started implementing support for running SIMATIC AX code on a software controller. The changes should be shipped rather soon. This should then also support using ax2tia libraries on such a runtime.

**Reply by RauerJ** [reply]
@theBadT Do you know, if this implementation is already shipped?

**Reply by theBadT** [reply]
With the latest STC in version 3.x, a new target swcpu has been introduced. You may use this to check whether the requirement is satisfied.
